pkg/config/testdata/TestLoadConfigSuccess/config.golden.yaml

```diff
@@ -175,3 +175,4 @@ plugins:
     incomingWebhook:
         enabled: false
         port: 0
+        inClusterBaseURL: ""
```

Comment:
Does that mean "In-cluster Base URL", or "Incoming cluster Base URL". I think this is used in slack buttons to go to ArgoCD dashbord, maybe better to state like "publicClusterBaseURL" or something.